docs/development-guide.md

```diff
@@ -17,6 +17,13 @@ it is **highly** encouraged to:
    [install](https://golang.org/doc/install) it on your system.
 * Setup the [GOPATH](http://www.g33knotes.org/2014/07/60-second-count-down-to-go.html)
    environment.
+* Ceph-CSI uses the native Ceph libaries through the [go-ceph
+   package](https://github.com/ceph/go-ceph). It is required to install the
+   Ceph C headers in order to compile Ceph-CSI. The packages are called
+   `libcephfs-devel`, `librados-devel` and `librbd-devel` on many Linux
```

Comment:
I think we don't need to point to go-ceph project here as it pointer to do `go get` `build` `tests` and all, we can just mention the packages required as told in go-ceph
ex
```text
The native RADOS library and development headers are expected to be installed.

On debian systems (apt):

libcephfs-dev librbd-dev librados-dev
On rpm based systems (dnf, yum, etc):

libcephfs-devel librbd-devel librados-devel
```

Reply:
I think the reference is helpful, just as the pointer to install Go is. In case there are additional dependencies for go-ceph later on (or for other OS'es), there is no need to update this text.